jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -416,6 +435,13 @@ ecma_builtin_typedarray_prototype_filter (ecma_value_t this_arg, /**< this objec
       goto cleanup;
     }
 
+    if (ecma_arraybuffer_is_detached (info_p->array_buffer_p))
+    {
+      ecma_free_value (call_value);
+      ret_value = ecma_raise_type_error (ECMA_ERR_MSG ("ArrayBuffer has been detached"));
```

Comment:
no assignment is needed, since `ret_value` is initialized to `ECMA_VALUE_ERROR`, just simply raise the error.